src/batched-json.js

```diff
@@ -65,9 +65,20 @@ export function batchFetchJsonFor(
           'ampViewerAuthToken': opt_token,
         };
       }
-      return xhr.fetchJson(data.xhrUrl, data.fetchOpt);
+      return Promise.all([
+        xhr.fetchJson(data.xhrUrl, data.fetchOpt),
+        Promise.resolve(data.stripPrefix),
+      ]);
+    })
+    .then(([res, stripPrefix]) => {
+      if (!stripPrefix) {
+        return res.json();
+      } else {
+        return res
+          .text()
+          .then(txt => parseJson(txt.slice(stripPrefix.length)));
```

Comment:
We need to check if the JSON is prefixed before stripping it.

Reply:
You don't think it is fair to assume that if the developer explicitly placed a `strip-prefix` attr then it will definitely be there?